hcloud/volume.go

```diff
@@ -21,12 +21,18 @@ type Volume struct {
 	Server      *Server
 	Location    *Location
 	Size        int
+	Format      *string
 	Protection  VolumeProtection
 	Labels      map[string]string
 	LinuxDevice string
 	Created     time.Time
 }
 
+const (
+	VolumeFormatExt4 = "ext4"
+	VolumeFormatXfs  = "xfs"
+)
```

Comment:
Do you think it's worth using a custom type?

Reply:
I decided against it because `VolumeCreateOpts` already used a string pointer for that property, so we would either have to make it inconsistent or a breaking change.